apps/porting/src/dns_adapter.c

```diff
@@ -8,7 +9,7 @@
 
 typedef struct {
     const char* id;
-    uint32_t resolved_addr;
+    uint32_t resolved_addrs[NABTO_DNS_RESOLVED_IPS_MAX];
```

Comment:
resolved_addrs not set to null, undefined behavior as the array is copied into unabto